install_config/advanced_ldap_configuration/configuring_extended_ldap_attributes.adoc

```diff
@@ -0,0 +1,181 @@
+= Configuring Extended LDAP Attributes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+This topic builds upon
+link:../advanced_ldap_configuration/ssd_for_ldap_failover.html[Setting up SSSD
+for LDAP Failover] and
+link:../advanced_ldap_configuration/configuring_form_based_authentication.html[Configuring
+Form-Based Authentication] and focuses on configuring extended Lightweight
+Directory Access Protocol (LDAP) attributes.
+
+[[configuring-extended-ldap-attributes-prerequisites]]
+== Prerequisites
+
+* SSSD 1.12.0 or later. This is available on Red Hat Enterprise Linux 7.0 and
+later.
+* mod_lookup_identity 0.9.4 or later.
+** This is not yet available on any version of Red Hat Enterprise Linux 7, but the
+Red Hat Package Managers (RPMs) for this platform are
+https://copr.fedorainfracloud.org/coprs/adelton/identity_demo/[available for
+upstream] until they arrive in Red Hat Enterprise Linux.
+
```

Comment:
We don't write "Red Hat Package Managers (RPMs)" anywhere. Rewrite it to:

"The required version is not yet available on any version of Red Hat Enterprise Linux, however compatible packages (RPMs) are available from upstream."

(Also: for->from when talking about upstream, which I realize I also typoed in my original blog)